.eslintrc.js

```diff
@@ -885,8 +862,10 @@ module.exports = {
      * TSVB overrides
      */
     {
-      files: ['src/legacy/core_plugins/metrics/**/*.js'],
-      excludedFiles: 'src/legacy/core_plugins/metrics/index.js',
```

Comment:
:information_source: We have an exclude at that one specific import, where I think it belongs, not inside the eslintrc.